BackofficeBundle/Controller/ContentController.php

```diff
@@ -107,11 +107,7 @@ public function newAction(Request $request, $contentType)
         if ($this->handleForm($form, $message, $content)) {
             $this->dispatchEvent(ContentEvents::CONTENT_CREATION, new ContentEvent($content));
 
-            return $this->redirect(
-                $this->generateUrl('open_orchestra_backoffice_content_form', array(
-                    'contentId' => $content->getContentId()
-                ))
-            );
+            return $this->render('BraincraftedBootstrapBundle::flash.html.twig');
```

Comment:
I'm not sure to understand: when a content is created we now see a page with only a flashbag ?